roles/servicetelemetry/defaults/main.yml

```diff
@@ -1,39 +1,57 @@
 ---
 # set default enablement
-metrics_enabled: true
-events_enabled: false
-high_availability_enabled: false
-storage_ephemeral_enabled: false
-graphing_enabled: false
-
-# default values
-prometheus_storage_class: ""
-prometheus_storage_resources: {}
-prometheus_storage_selector: {}
-prometheus_pvc_storage_request: 20G
-
-alertmanager_storage_class: ""
-alertmanager_storage_resources: {}
-alertmanager_storage_selector: {}
-alertmanager_pvc_storage_request: 20G
-
-elasticsearch_node_count: 1
-elasticsearch_pvc_storage_request: 20Gi
-elastic_user: elastic
-
-qdr_deployment_size: 1
-
-smartgateway_servicemonitor_spec_endpoints_interval: 10s
-
-graphing_spec_ingress_enabled: false
-graphing_spec_config_security_admin_password: secret
-graphing_spec_config_security_admin_user: root
-graphing_spec_config_auth_disable_signout_menu: false
+metrics:
+  enabled: true
+    smartgateway:
+      servicemonitor:
+        endpoints_interval: 10s
+    prometheus:
+      storage_class: ""
+      storage_resources: {}
+      storage_selector: {}
+      pvc_storage_request: 20G
+
+events:
+  enabled: false
+  elasticsearch:
+    node_count: 1
+    pvc_storage_request: 20Gi
+
+high_availability:
+  enabled: false
+
+storage:
+  ephemeral:
+    enabled: false
+
+graphing:
+  enabled: false
+
+alerting:
+  alertmanager:
+    storage_class: ""
+    storage_resources: {}
+    storage_selector: {}
+    pvc_storage_request: 20G
+
+
+transport:
+  qdr:
+    deployment_size: 1
+
+
+graphing:
+  grafana:
+    ingress_enabled: false
+    admin_password: secret
+    admin_user: root
+    disable_signout_menu: false
```

Comment:
I'm pretty sure this entire section needs to go under a top-level `servicetelemetry_defaults` dict - this is how the magic works in the vars/main.yaml. [See where your vars/main.yml line 6 uses `servicetelemetry_defaults`](https://github.com/infrawatch/service-telemetry-operator/pull/119/files#diff-6d0d7b6f88656c55602ecc09636b1337R6) ?? That var should be defined here in your defaults/main.yml

Reply:
Ah yes! I knew that actually, but it seems I made a mod somewhere and removed it. Nice catch! I haven't tested this yet, but I'm sure I'd have been confused for some time :) Will fix!

Reply:
Oh I know what I did. I had that on my _other_ branch that I was experimenting with, then I ported the logic to a separate branch, and I forgot to add that whe I did that.